src/main/java/com/crowdin/cli/commands/actions/DownloadReviewedSourcesAction.java

```diff
@@ -0,0 +1,256 @@
+package com.crowdin.cli.commands.actions;
+
+import com.crowdin.cli.client.CrowdinProjectFull;
+import com.crowdin.cli.client.ProjectClient;
+import com.crowdin.cli.commands.NewAction;
+import com.crowdin.cli.commands.Outputter;
+import com.crowdin.cli.commands.functionality.FilesInterface;
+import com.crowdin.cli.commands.functionality.ProjectFilesUtils;
+import com.crowdin.cli.commands.functionality.PropertiesBeanUtils;
+import com.crowdin.cli.commands.functionality.SourcesUtils;
+import com.crowdin.cli.commands.functionality.TranslationsUtils;
+import com.crowdin.cli.properties.PropertiesWithFiles;
+import com.crowdin.cli.utils.PlaceholderUtil;
+import com.crowdin.cli.utils.Utils;
+import com.crowdin.cli.utils.console.ConsoleSpinner;
+import com.crowdin.client.sourcefiles.model.Branch;
+import com.crowdin.client.sourcefiles.model.BuildReviewedSourceFilesRequest;
+import com.crowdin.client.sourcefiles.model.FileInfo;
+import com.crowdin.client.sourcefiles.model.ReviewedStringsBuild;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import static com.crowdin.cli.BaseCli.RESOURCE_BUNDLE;
+import static com.crowdin.cli.utils.console.ExecutionStatus.ERROR;
+import static com.crowdin.cli.utils.console.ExecutionStatus.WARNING;
+
+public class DownloadReviewedSourcesAction implements NewAction<PropertiesWithFiles, ProjectClient> {
```

Comment:
How about tests?

Reply:
Let's first do manual testing first. Otherwise it might lead to double work where I need to change both implementation and tests.